lib/screens/login_page.dart

```diff
@@ -31,7 +31,7 @@ class _LoginPageState extends State<LoginPage> {
                 onDetect: (List<Barcode> barcodes, Uint8List? image) {
                   setState(() {
                     scannedQRCode =
-                        barcodes.isNotEmpty ? barcodes[0].rawValue : null;
+                    barcodes.isNotEmpty ? barcodes[0].rawValue : null;
```

Comment:
```suggestion
                         barcodes.isNotEmpty ? barcodes[0].rawValue : null;
```
Add whitespace indentation